sdk/servicebus/azure-messaging-servicebus/CHANGELOG.md

```diff
@@ -1,7 +1,14 @@
 # Release History
 
-## 7.0.0-beta.2 (Unreleased)
-
+## 7.0.0-beta.2 (2020-05-07)
+
+- Add support for receiving messages from specific sessions
+- Add support for processing messages from multiple sessions
```

Comment:
Why does one say `receiving` and the other `processing`?

Reply:
Grammarly told me to try another verb. LOL.

Reply:
I would expect both the bullet points to use the same verb